mediorum/server/server.go

```diff
@@ -148,6 +148,18 @@ func New(config MediorumConfig) (*MediorumServer, error) {
 	basePath.Use(middleware.Recover())
 	basePath.Use(middleware.CORS())
 
+	// TODO: Use middleware to cache whenever content is streamed, except if it's premium content.
+	// See Johannes's previous PR for reference. From CN:
+	// If content is gated, set cache-control to no-cache.
+	// Otherwise, set the CID cache-control so that client caches the response for 30 days.
+	// The contentAccessMiddleware sets the req.contentAccess object so that we do not
+	// have to make another database round trip to get this info.
```

Comment:
Not sure if middleware is simpler than just explicitly calling a helper function in a handler.  Granted this is just a comment anyway, but just sayin for future reference.